test/e2e/performanceprofile/functests/utils/node_inspector/inspector.go

```diff
@@ -94,7 +127,7 @@ func getDaemonPodByNode(node *corev1.Node) (*corev1.Pod, error) {
 // ExecCommand executing the command on a daemon pod of the given node
 func ExecCommand(ctx context.Context, node *corev1.Node, command []string) ([]byte, error) {
 	// Ensure the node inspector is running
-	ok, err := isRunning(testclient.DataPlaneClient, testutils.NodeInspectorNamespace, testutils.NodeInspectorName)
+	ok, err := isRunning()
 	if err != nil || !ok {
```

Comment:
In case of `ok` and `err == nil` it returns `nil, nil` which means the caller if `ExecCommand` will continue as usual since there was not error reported